### ISSUE TITLE
Implement option to decode the name of the file

### ISSUE THREAD

**Issue by inkedsaber** [inkedsaber/perl-zip1#7]
Add option ('u/unicode' or something like that) to decode name of the file.
Example: 
```
~:$>zip1 -td /path/to/zip1
OK: 39131__34892__34909__21205_-__12450__12512__12493__12472__12450.zip1 -> 39131__34892__34909__21205_-__12450__12512__12493__12472__12450.zip
OK: name of the file was decoded: 39131__34892__34909__21205_-__12450__12512__12493__12472__12450.zip -> 飛行衝動 - アムネジア.zip
```

**Comment by inkedsaber** [comment]
Done

**Comment by inkedsaber** [comment]
Issue #7 is closed (ref: [bdf0c92](https://github.com/inkedsaber/perl-zip1/commit/bdf0c92103f3b4d337790935bc5f0d11a0cf1a49))
